pkg/stagemanager/stage_manager.go

```diff
@@ -274,7 +272,7 @@ func (stm *StageManager) runStartStage() {
 
 // after start, already working (accepting request)
 func (stm *StageManager) AppendAfterStartStage(f func(Application)) *StageManager {
-	if f == nil || stm.started {
+	if f == nil || stm.state == Running {
```

Comment:
ditto.

Reply:
done, but I actually found that this function is not called.

Reply:
yeah, it's not used in this MOSN core project, but it may used in other MOSN based project. eg, we used it in ant group internal.